tech/languages/csharp/csharp-ide.md

```diff
@@ -22,29 +22,30 @@ JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core
 
 [jetbrains.com/rider](http://jetbrains.com/rider)
 
-## MonoDevelop
-
-_Mono only_
+## Eclipse IDE (with plugin)
 
-MonoDevelop provides the best C# experience on Mono, the open-source multiplatform implementation of the .NET Framework.
+_.NET Core and Mono_
 
-[monodevelop.com](http://www.monodevelop.com)
+You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE. Relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
 
 ```
-$ sudo dnf install monodevelop
+$ sudo dnf install eclipse eclipse-mpc
 ```
+then install the aCute extension from [Eclipse Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
```

Comment:
`How?`
Isn't there supposed to be hint- something like 'follow this guide' perhaps?

Reply:
I've just noticed you've only changed the order of blocks. However, since you're updating it, could we enhance it too?

Reply:
I don't know how, I have **failed** to actually use eclipse with c#. It imho sucks =)
On the plugin website they say "drag this thing into eclipse ide and it will install" which doesn't work for me. I had to find it in the eclipse ide's search which was weirdly hidden, and even then, I was unable to edit any C# files or netcore projects.

That said, that's just me, it's supposed to be "an awesome ide that actually works," so I'm not gonna argue with that or tell people to not try on their own.

Reply:
Come to think of it, maybe we should at least mention it does not have to work, or there might me some difficulties.